aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -77,6 +77,25 @@ module aptos_std::simple_map {
         vector::push_back(&mut map.data, Element { key, value });
     }
 
+    public fun to_vec<Key: store, Value: store>(
+        map: SimpleMap<Key, Value>): vector<Element<Key, Value>> {
+        let SimpleMap { data } = map;
+        data
+    }
+
+    public fun split_element<Key: store, Value: store>(e: Element<Key, Value>): (Key, Value) {
+        let Element { key, value} = e;
+        (key, value)
+    }
+
+    public inline fun destroy<Key: store, Value: store>(
```

Comment:
I wonder if we should provide a second destroy function where both Key and Value have drop. This would be simpler to use as it's not inline + doesn't require the drop function.

Reply:
If key/value have drop you don't need to destroy in the first place